releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -2966,13 +3049,12 @@
     ],
     "sigs": [
       "cluster-lifecycle"
-    ],
-    "is_mapped": true
+    ]
   },
   "125168": {
     "commit": "d8b33e32b16a8e393f9e174af92b19302c729a98",
     "text": "job-controller: the `JobReadyPods` feature flag has been removed (deprecated since v1.31)",
-    "markdown": "Job-controller: the `JobReadyPods` feature flag has been removed (deprecated since v1.31) ([#125168](https://github.com/kubernetes/kubernetes/pull/125168), [@kaisoz](https://github.com/kaisoz)) [SIG Apps]",
+    "markdown": "Job-controller: the `JobReadyPods` feature flag has been removed (deprecated since v1.31) ([#125168](https://github.com/kubernetes/kubernetes/pull/125168), [@kaisoz](https://github.com/kaisoz))",
```

Comment:
```suggestion
    "markdown": "Job-controller: the `JobReadyPods` feature flag has been removed (deprecated since v1.31) ([#125168](https://github.com/kubernetes/kubernetes/pull/125168), [@kaisoz](https://github.com/kaisoz)) [SIG Apps]",
```